### ISSUE TITLE
Thing action support

### ISSUE THREAD

**Issue by spacemanspiff2007** [spacemanspiff2007/HABApp#353]
Thing actions are now available through [#926](https://github.com/openhab/openhab-core/issues/926).
Add support.

**Comment by lukiep** [comment]
Would this potentially include the ability to use eg  equivalent of NotificationAction.sendNotification  ?  I understand it is not currently possible in habapp

**Comment by spacemanspiff2007** [comment]
Correct. Currently you have to use a wrapper item and if needed native openHAB rules.
If I implement this you can call the actions directly from HABApp.

**Comment by dimka-dk** [comment]
Hey @spacemanspiff2007 - I would like to use thing actions from https://www.openhab.org/addons/bindings/energidataservice/#thing-actions - what's the progress with this Feature Request ?
